src/main/java/com/github/pgasync/impl/PgConnectionPool.java

```diff
@@ -142,9 +143,10 @@ public void close() throws Exception {
 
     @Override
     public Observable<Connection> getConnection() {
-        return Observable.<Connection>create(subscriber -> {
+        return Observable.<Connection>unsafeCreate(subscriber -> {
```

Comment:
when creating Observables manually the implementer should take care of unsubscription which I cannot find in this class

Reply:
I know, that's why it needs deeper refactoring. I've just changed method name to indicate this.